active_learning/meta_path_selector.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from typing import List
+from util.datastructures import MetaPath
+from .hypothesis import Hypothesis
+from abc import ABC, abstractmethod
+
+class AbstractMetaPathSelector(ABC):
+    """
```

Comment:
Method comments have to be in the method

Reply:
this comment describes what the class does

Reply:
Oh sorry